contracts/LRTDepositPool.sol

```diff
@@ -319,6 +319,44 @@ contract LRTDepositPool is ILRTDepositPool, LRTConfigRoleChecker, PausableUpgrad
         }
     }
 
+    /// @notice Transfers all specified assets lying in this DepositPool to a node delegator
+    /// @dev only callable by LRT Manager
+    /// @param ndcIndex Index of NodeDelegator contract address in nodeDelegatorQueue starting at zero.
+    /// @param assets List of asset addresses
+    function transferAssetsToNodeDelegator(
+        uint256 ndcIndex,
+        address[] calldata assets
+    )
+        external
+        override
+        nonReentrant
+        onlyLRTOperator
+    {
+        address nodeDelegator = nodeDelegatorQueue[ndcIndex];
```

Comment:
I guess this is okay, since solidity now checks that the lookup is inside the array length. But still, maybe a zero address check on the nodeDelegator just for sanity?

Reply:
Good idea. I'll add